PyPoE/cli/exporter/wiki/parsers/itemconstants.py

```diff
@@ -0,0 +1,60 @@
+"""
+Overview
+===============================================================================
+
++----------+------------------------------------------------------------------+
+| Path     | PyPoE/poe/constants.py                                           |
++----------+------------------------------------------------------------------+
+| Version  | 1.0.0a0                                                          |
++----------+------------------------------------------------------------------+
+| Revision | $Id$                  |
++----------+------------------------------------------------------------------+
+| Author   | angelic_knight                                                   |
++----------+------------------------------------------------------------------+
+
+Description
+===============================================================================
+
+Global constants the item exporter such as lists of items to exclude from exporting.
+
+Agreement
+===============================================================================
+
+See PyPoE/LICENSE
+
+Documentation
+===============================================================================
+"""
+
+# =============================================================================
+# Imports
+# =============================================================================
+
+# Python
+
+# 3rd-party
+
+# self
+
+# =============================================================================
+# Globals
+# =============================================================================
+
+
+MAPS_IN_SERIES_BUT_NOT_ON_ATLAS = {
+    'Metadata/Items/Maps/MapWorldsHarbingerUber',
+    'Metadata/Items/Maps/MapWorldsHarbingerHigh',
+    'Metadata/Items/Maps/MapWorldsHarbingerMid',
+    'Metadata/Items/Maps/MapWorldsHarbingerLow',
+    'Metadata/Items/Maps/MapWorldsPhoenix',
+    'Metadata/Items/Maps/MapWorldsChimera',
+    'Metadata/Items/Maps/MapWorldsHydra',
+    'Metadata/Items/Maps/MapWorldsMinotaur',
+}
+
+MAPS_TO_SKIP_COLORING_AND_COMPOSITING = {
+    'Metadata/Items/Maps/MapWorldsHarbingerUber',
+    'Metadata/Items/Maps/MapWorldsHarbingerHigh',
+    'Metadata/Items/Maps/MapWorldsHarbingerMid',
+    'Metadata/Items/Maps/MapWorldsHarbingerLow',
+}
```

Comment:
New line at the end of any Python file. (Unless your formatter had a hiccup?)

Reply:
VSCode didn't add it and neither did I.
Manually added one.